src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/StateSynchronizationMenuItems.cs

```diff
@@ -7,13 +7,42 @@
 using System.IO;
 using System.Collections.Generic;
 using UnityEngine.SceneManagement;
+using System.Collections;
 
 namespace Microsoft.MixedReality.SpectatorView.Editor
 {
     public static class StateSynchronizationMenuItems
     {
         private static IEqualityComparer<IAssetCache> assetTypeComparer = new AssetCacheTypeEqualityComparer();
 
+        private const string disableSpectatorViewPreBuild = "Spectator View/Disable Spectator View Pre-build steps";
+        public static bool DisablePreBuildSteps
+        {
+            get
+            {
+                return PlayerPrefs.GetInt(disableSpectatorViewPreBuild, 0) > 0;
+            }
+            private set
+            {
+                PlayerPrefs.SetInt(disableSpectatorViewPreBuild, value ? 1 : 0);
+                PlayerPrefs.Save();
+            }
+        }
+
+        private const string disableUpdatingMenuItem = "Spectator View/Disable updating asset caches when building";
```

Comment:
>Disable updating asset caches when building [](start = 71, length = 43)

Menu items should be in Title Case, so should be "Disable Updating Asset Caches When Building"